### PR TITLE
PMM-6797 Add haproxy service into pmm-admin.md.

### PR DESCRIPTION
**Pull request by JiriCtvrtka**

### CODE REVIEW

**Comment by JiriCtvrtka** [comment]
Order of flags is same like in pmm-admin output.
Please is target branch "main" OK?

**Review by PaulJacobs-percona — APPROVED** [review]
Nice work

File: docs/details/commands/pmm-admin.md [hunk 1]
```diff
@@ -18,6 +18,8 @@
 
 `pmm-admin add DATABASE [FLAGS] [NAME] [ADDRESS]`
 
+`pmm-admin add haproxy [FLAGS] [NAME]` (CAUTION: Technical preview feature)
```
Comment (rnovikovP) [inline docs/details/commands/pmm-admin.md]: not a Technical Preview

File: docs/details/commands/pmm-admin.md [hunk 2]
```diff
@@ -508,6 +510,58 @@ PMM communicates with the PMM Server via a PMM agent process.
...
+    : HAProxy password.
+
+    `--scheme=SCHEME`
+    : Scheme to generate URI to exporter metrics endpoints (http or https).
+
+    `--metrics-path=METRICS-PATH`
+    : Path under which metrics are exposed, used to generate URI (default: /metrics).
+
+    `--listen-port=LISTEN-PORT`
+    : Listen port of external exporter for scraping metrics (Required).
```
Comment (rnovikovP) [inline docs/details/commands/pmm-admin.md:539]:
```suggestion
    : Listen port of haproxy exposing the metrics for scraping metrics (Required).
```

Comment (JiriCtvrtka) [reply docs/details/commands/pmm-admin.md]: @rnovikovP Yes, here https://github.com/percona/pmm-admin/blob/PMM-6797-cli-for-haproxy/commands/management/add_haproxy.go#L130:L130, going to change it to same message like here. Only another occurrence what I know is in pmm inventory in grafana. Where you can see external exporter which was added with haproxy service. Is it OK?

**Review by rnovikovP — APPROVED** [review]

**Review by PaulJacobs-percona — APPROVED** [review]

**Pull request merged**